contrib/backporting/cherry-pick

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
Missing copyright header

Reply:
We don't really have it in all the small scripts. I'd leave it out. Should be covered anyway from the top level LICENSE file.

Reply:
OK.

Reply:
Btw, we could at some point just use SPDX ids like in the kernel.